dbms/src/Debug/dbgKVStore/dbgFuncInvestigator.cpp

```diff
@@ -143,16 +143,16 @@ void dbgFuncFindKey(Context & context, const ASTs & args, DBGInvoker::Printer ou
                 "start_col, [, start_col2, ..., end_col1, end_col2, ...]",
                 arg_size);
         }
-        size_t handle_column_size = table_info.is_common_handle ? table_info.getPrimaryIndexInfo().idx_cols.size() : 1;
         auto key_size = arg_size / 2;
         std::vector<Field> start_field;
         start_field.reserve(key_size);
         std::vector<Field> end_field;
         end_field.reserve(key_size);
 
-        for (size_t i = 0; i < handle_column_size; i++)
+        const auto & pk_index = table_info.getPrimaryIndexInfo();
+        for (size_t i = 0; i < pk_index.idx_cols.size(); i++)
```

Comment:
```suggestion
        const auto & pk_index = table_info.getPrimaryIndexInfo();
        for (const auto & idx_col : pk_index.idx_cols)
        {
            auto & column_info = table_info.columns[idx_col.offset];
        }
```

Reply:
We can not do that because line 157 require the `i` to get the offset of input `args`